### DISCUSSION TITLE
DPC Alternative? 💀

### DISCUSSION THREAD

**Discussion by GreenMystic** [BinTianqi/OwnDroid#59] [Q&A] [answered]
dev do you have alternative for using setting that required devices owner? 💀

bcs cost to switch is too high 💀
(need removing account + user)
i can't remove users (work profile)
i need it for my daily 💀

**Comment by BinTianqi** [comment, marked as answer]
> device owner

> work profile

When there is a work profile on your device, you can't activate device owner. When a device owner is activated, you can't create a work profile. 

Read the docs to get more info: https://owndroid.pages.dev

**Reply by GreenMystic** [reply]
what the?
in the past island or insular able to create work profile before dpc activated
why is impossible now?
android update? 💀
wtf I didn't do any OTA Update since dinosaur age 💀

see:
[Island / Insular Setup](https://island.oasisfeng.com/setup.html)
[Limitations of managed mainland](https://island.oasisfeng.com/#managed-mainland)

**Reply by BinTianqi** [reply]
> island or insular able to create work profile before dpc activated
> why is impossible now?

I mean you can't create a work profile *after* dpc activated

**Reply by GreenMystic** [reply]
i confused 🥴
in the past with island / insular i could activate dpc even there is work profile 
but why now it's impossible? 💀
(since A11?)

**Reply by BinTianqi** [reply]
First, dpc can't be activated when there are more than 1 user. 
Second, actually, work profile is a user.

So, you can't activate OwnDroid when a work profile exist on your device.

**Reply by GreenMystic** [reply]
> First, dpc can't be activated when there are more than 1 user. Second, actually, work profile is a user.
> 
> So, you can't activate OwnDroid when a work profile exist on your device.

i no longer understand how android works 🥴

maybe different app can have different result 
maybe different android can have different result as well
(in the past on A10 + A11 i can activate devices owner on island after work profile created so island can manage main profile as well)

Android + Google + Manufactur The Horsemen Of Annoyance 🗿
